charts/unleash/values.yaml

```diff
@@ -24,90 +24,82 @@ autoscaling:
 # a shared secret for secure communication with a backend service.
 configMaps: {}
 #  index.js:
-#    mountPath: /index.js
+#    mountPath: /unleash/index.js
```

Comment:
Working directory in a container is `/unleash`, so we have to put this file to `/unleash/index.js`.